receiver/filelogreceiver/README.md

```diff
@@ -36,6 +36,7 @@ Tails and parses logs from files.
 | `include_file_record_number`            | `false`                              | Whether to add the record number in the file as the attribute `log.file.record_number`.                                                                                                                                                                    |
 | `poll_interval`                       | 200ms                                | The [duration](#time-parameters) between filesystem polls.                                                                                                                                                                                                      |
 | `fingerprint_size`                    | `1kb`                                | The number of bytes with which to identify a file. The first bytes in the file are used as the fingerprint. Decreasing this value at any point will cause existing fingerprints to forgotten, meaning that all files will be read from the beginning (one time) |
+| `max_surge_size`                      |                                      | The maximum surge size of a log file to read. log entries will be skipped if it is larger than `max_surge_size`. Protects against lagging behind when reading large log files.                                                                                  |
```

Comment:
It doesn't have a default value, or is it missing documentation ?

Reply:
default value is zero value, will update